editor/block-mover/test/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { BlockMover } from '../';
+
+describe( 'BlockMover', () => {
+	describe( 'basic rendering', () => {
+		it( 'should render two IconButton components with the following props', () => {
+			const blockMover = shallow( <BlockMover /> );
+			expect( blockMover.hasClass( 'editor-block-mover' ) );
+
+			const moveUp = blockMover.childAt( 0 );
+			const moveDown = blockMover.childAt( 1 );
+			expect( moveUp.type().name ).to.equal( 'IconButton' );
+			expect( moveDown.type().name ).to.equal( 'IconButton' );
+			expect( moveUp.props() ).to.include( {
+				className: 'editor-block-mover__control',
+				onClick: undefined,
```

Comment:
More specifically, it interprets the assertion without making any expectations about the value of `onClick`, so the test would behave the same if the `onClick` property weren't included at all.

```js
expect( moveUp.props() ).to.include( {
	className: 'editor-block-mover__control',
	icon: 'arrow-up-alt2',
	label: 'Move 2 blocks from position 1 up by one place',
	'aria-disabled': undefined,
} );
```

Which, of course, is probably not what we want.